src/binding.cpp

```diff
@@ -302,11 +302,17 @@ NAN_METHOD(render_file_sync) {
   NanReturnValue(NanNew<Boolean>(result == 0));
 }
 
+NAN_METHOD(libsass_version) {
+  NanScope();
+  NanReturnValue(NanNew<String>(libsass_version()));
```

Comment:
Yeah cool that's what I figured but I missed it in a quick grep of libsass. I see it now :+1: